install.php

```diff
@@ -9,7 +9,8 @@
 if(!defined('DOKU_CONF')) define('DOKU_CONF',DOKU_INC.'conf/');
 if(!defined('DOKU_LOCAL')) define('DOKU_LOCAL',DOKU_INC.'conf/');
 
-require_once(DOKU_INC.'inc/PassHash.class.php');
+// load and initialize the core system
+require_once(DOKU_INC.'inc/init.php');
```

Comment:
Was there a reason why not the whole init.php was included before?
I guess the needed line is the autoload of the vendor libraries (line 194)

https://github.com/splitbrain/dokuwiki/blob/master/inc/init.php#L194

snippet of init.php:
```php
// load libraries
require_once(DOKU_INC.'vendor/autoload.php');
require_once(DOKU_INC.'inc/load.php');
```